otherlibs/site/src/plugins/linker/toplevel/linker.ml

```diff
@@ -0,0 +1,13 @@
+open Topdirs [@@ocaml.warning "-33"]
+open Toploop [@@ocaml.warning "-33"]
```

Comment:
Why do we need these opens?

Reply:
> Why do we need these opens?

Before 4.13.0, loadfile is in Topdirs. After it is in Toploop. We either need some kind of conditional compilation, or else just open both and let the compiler find it. Is there a better way?

Reply:
No, this seems adequate. Could you add a comment in the source explaining this however

Reply:
> No, this seems adequate. Could you add a comment in the source explaining this however

Will do.